README.md

```diff
@@ -340,6 +340,14 @@ The following settings are optional and allow you to further configure your clus
     ```
```

Comment:
Would you please show testing in the description for the case where the new settings are unset?

Reply:
Sure thing. I just updated that case in description. Thanks